### PR TITLE
Added marshmallow-sqlalchemy recipe

### PR DESCRIPTION
**Pull request by frol**
SQLAlchemy integration with the marshmallow (de)serialization library.

https://github.com/marshmallow-code/marshmallow-sqlalchemy

### CODE REVIEW

File: recipes/marshmallow-sqlalchemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - marshmallow >=2.0.0
+    - sqlalchemy >=0.9.7
+
+test:
+  imports:
+    - marshmallow_sqlalchemy
+
+about:
+  home: https://github.com/marshmallow-code/marshmallow-sqlalchemy
+  license: MIT
```
Comment (jakirkham) [inline recipes/marshmallow-sqlalchemy/meta.yaml]: Could optionally add `license_family: MIT`.

File: recipes/marshmallow-sqlalchemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - marshmallow >=2.0.0
+    - sqlalchemy >=0.9.7
```
Comment (jakirkham) [inline recipes/marshmallow-sqlalchemy/meta.yaml]: Please drop anything from `build` that is not strictly a requirement at build time. Probably just need `python` and `setuptools`.

File: recipes/marshmallow-sqlalchemy/meta.yaml (hunk 1 above)
Comment (frol) [inline recipes/marshmallow-sqlalchemy/meta.yaml]: Updated.

File: recipes/marshmallow-sqlalchemy/meta.yaml (hunk 2 above)
Comment (frol) [inline recipes/marshmallow-sqlalchemy/meta.yaml]: Updated.

**Pull request merged**